Project.toml

```diff
@@ -21,9 +21,6 @@ AssetRegistry = "0.1"
 Blink = "0.12"
 FFTW = "1.0"
 Interact = "0.10"
-KomaMRICore = "0.8"
-KomaMRIFiles = "0.8"
-KomaMRIPlots = "0.8"
```

Comment:
Revert

Reply:
Resolved in [7f1cc52](https://github.com/JuliaHealth/KomaMRI.jl/pull/184/commits/7f1cc52925d3be8803c4ff0c95c1e06f391b7c83)